config/example/config.txt

```diff
@@ -197,25 +167,20 @@ DEFAULT_NO_VOTE
 ALLOW_ANTAG_HUD
 
 ## If ghosts use antagHUD they are no longer allowed to join the round.
-ANTAG_HUD_RESTRICTED
+# ANTAG_HUD_RESTRICTED
 
 ## allow AI job
 ALLOW_AI
 
-## Disable respawning
-# NORESPAWN
-
-## set a respawn time (in minutes)
-# RESPAWN_TIME 5
 
-## set a message to give to players when they respawn
-# RESPAWN_MESSAGE Remember to play a different character or something!
+## disable abandon mob
+# NORESPAWN
 
 ## disables calling del(src) on newmobs if they logout before spawnin in
 # DONT_DEL_NEWMOB
 
 ## set a hosted by name for unix platforms
-HOSTEDBY yournamehere
+HOSTEDBY Spades

```

Comment:
This should probably be changed on the live server.

Reply:
Don't request changes on a PR that has nothing to do with your requested changes